cmd/update.go

```diff
@@ -153,42 +141,16 @@ func setupUpdateCommand() *cli.Command {
 
 func runUpdateSuperNodeRemoteSubCommand(ctx context.Context, config *configs.Config) (err error) {
 
-	// Validate config
-	if len(updateConfigFlags.RemoteIP) == 0 {
-		return fmt.Errorf("--ssh-ip IP address - Required, SSH address of the remote host")
-	}
-
 	// Connect to remote
-	var client *utils.Client
-	log.WithContext(ctx).Infof("Connecting to remote host -> %s:%d...", updateConfigFlags.RemoteIP, updateConfigFlags.RemotePort)
-
-	if len(updateConfigFlags.SSHKey) == 0 {
-		username, password, _ := utils.Credentials(updateConfigFlags.RemoteUser, true)
-		client, err = utils.DialWithPasswd(fmt.Sprintf("%s:%d", updateConfigFlags.RemoteIP, updateConfigFlags.RemotePort), username, password)
-	} else {
-		username, _, _ := utils.Credentials(updateConfigFlags.RemoteUser, false)
-		client, err = utils.DialWithKey(fmt.Sprintf("%s:%d", updateConfigFlags.RemoteIP, updateConfigFlags.RemotePort), username, updateConfigFlags.SSHKey)
-	}
+	client, err := prepareRemoteSession(ctx, config)
 	if err != nil {
-		return err
+		log.WithContext(ctx).Infof("failed to prepare remote session: %v", err)
```

Comment:
same - change to : 
log.WithContext(ctx).WithError(err).Error("Failed to prepare remote session")